frontend/src/container/GridGraphLayout/Graph/FullView/index.tsx

```diff
@@ -126,13 +135,12 @@ const FullView = ({
 		onFetchDataHandler();
 	}, [onFetchDataHandler]);
 
-	if (state.error && !state.loading) {
-		return (
-			<NotFoundContainer>
-				<Typography>{state.errorMessage}</Typography>
-			</NotFoundContainer>
-		);
-	}
+	useEffect(() => {
+		state.error &&
+			notification.error({
```

Comment:
Best place to notify user would be in the catch block and you can also avoid using`useEffect` cc @pal-sig  What do you think?

Reply:
Yes @jyash97 it is perfect place